lesson24/src/js/main.js

```diff
@@ -0,0 +1,48 @@
+const showModal = (target)  => {
+    target.classList.remove('hidden');
+    target.setAttribute('aria-modal', 'true');
+    target.setAttribute('role', 'dialog');
+    target.removeAttribute('aria-hidden');
+}
+
+const hideModal = (target)  => {
+    target.classList.add('hidden');
+    target.setAttribute('aria-hidden', 'true');
+    target.removeAttribute('aria-modal');
+    target.removeAttribute('role');
+}
+
+const toggleModal = () => {
+    const modalTarget = document.querySelector('[data-id="popup-modal"]');
+    const modalTriggerElements = document.querySelectorAll("[data-modal-toggle]");
+    const modalIsHidden = modalTarget.classList.contains("hidden");
+    
+    modalTarget.addEventListener('click',() => hideModal(modalTarget));
+    
+    modalTriggerElements.forEach((trigger) => {
+        trigger.addEventListener('click', () => {
+            modalIsHidden ? showModal(modalTarget) : hideModal(modalTarget);
+        });
+    });
+}
+
+const agreeCheckBox = document.querySelector('[data-id="agree_check"]');
+const agreeButton   = document.querySelector('[data-id="agree_button"]');
+const submitButton  = document.querySelector('[data-id="submit_button"]');
+
+agreeCheckBox.addEventListener('change' , () => {
+    agreeCheckBox.checked ? submitButton.disabled = false : submitButton.disabled = true;
```

Comment:
IMO
You can also write like this
```suggestion
    submitButton.disabled = agreeCheckBox.checked ? false : true;
```

Reply:
Thank you for your comment!
I didn't know you could write like this!
Thanks for the great learning.

Reply:
I fixed.
https://github.com/nakagawamai/js-lesson/pull/25/commits/f808c1af7e818969446aeb0a422bbad4242e2973